### ISSUE TITLE
Laravel fixes not working

### ISSUE THREAD

**Issue by roylaveto** [laravel-shift/workbench-issues#33]
**Desktop:**
 - OS: Mac
 - OS Version: Big Sur
 - App Version: 0.4.7

**Describe the bug**
The normal fixes run just fine. But the 'Laravel' fixes finish within 1 second and don't show any output. Maybe it's just me but it doesn't seem to be working no matter what I do.

**To Reproduce**
1. Run a Laravel fix (make sure there are things to fix)
2. Wait for the result

https://imgur.com/a/ReSCpc1

**Comment by jasonmccreary** [comment]
@Roy-Ernst, we can definitely work on the output. However, are you sure your project had models which should be "fixed"?

Just trying to distinguish if this is simply an output issue or if there was code it missed. If the latter, please provide a code snippet you expected this task to change.

**Comment by roylaveto** [comment]
Trying to run Model Date Casts.

There are 2 files with:
```
    /**
     * @var string[]
     */
    protected $dates = [
        'publish_at',
        'unpublish_at',
        'created_at',
        'updated_at',
    ];
```

But every Laravel fix has a 0-1 second time before completing, which is really odd. Other fixes take at least 5-10 seconds.

**Comment by jasonmccreary** [comment]
Some tasks use system tools underneath which might be slow, some are simply string manipulation which can be very fast. Runtime is just a metric. It's not an indication of change.

With that said, it should have changed the snippet above. Out of curiosity, where is the file located in your project structure?

**Comment by roylaveto** [comment]
Oh didn't know that. The more you know! Haha.

It is located in: [project]/galaxy/modules/WebsiteNews/Models/WebsiteNews.php
I added `"Galaxy\\": "galaxy/"` to the PSR-4 autoload for composer. Which made it check that folder as well for other fixes. So I know it does read the folder.

**Comment by jasonmccreary** [comment]
Ah, so that's likely the bug here, this task may not be respecting the custom folder path. We'll take a look.

**Comment by roylaveto** [comment]
Great! Let me know when it's fixed and I'll test it out!

**Comment by jasonmccreary** [comment]
This should be resolved in the latest release (0.5.1). Let us know if not, or you find any other tasks which may not be respecting your autoload paths.
